SH4RE/SH4RE/Messages/Components/MessageView.swift

```diff
@@ -9,77 +9,87 @@ import SwiftUI
 
 struct MessageView: View {
     let message: ChatMessage
-    @State var requestStatus: Int = 0
+    @State var requestStatus: Int = -1
```

Comment:
what is this status? maybe add enum if there's mult states?

Reply:
Going to skip this for now in interest of time. But ideally yes this would be an enum